pages/passwordPage.js

```diff
@@ -275,6 +275,8 @@ class PasswordPage extends Common {
          }
          const hash = await this._hash($setup_p1.val());
          await storage.set("user", "passwordHash", hash);
+         const passwordHash = await storage.get("user", "passwordHash");
+         storage.config = { encrypt: true, key: passwordHash };
```

Comment:
Could we just reuse `hash` above instead of getting it from the storage?

Reply:
It seemed to be necessary, but I tested without it again and it turned out to be fine.